content/api/vision/coordinates.md

```diff
@@ -47,9 +47,33 @@ robot, exactly 1 metre away.
 
 {{% added_in update="2018-jan" feature="spherical coordinates" %}}
 
-## Diagram
+### Diagram
 
 The following diagram shows the orientation of the Cartesian axes relative to
 the camera as well as the angles which describe the spherical coordinate space.
 
 ![A diagram showing the coordinate spaces](/img/api/coordinate-spaces.svg)
+
+## Orientation Coordinates
+
+Orientation represents the rotation of a marker around the x, y, and z axes.
+These can be accessed as follows:
+
+- `rot_x_radians` / `rot_x_degrees` - the angle of rotation in radians/degrees
+   counter-clockwise about the Cartesian x axis.
+- `rot_y_radians` / `rot_y_degrees` - the angle of rotation in radians/degrees
+   counter-clockwise about the Cartesian y axis.
+- `rot_z_radians` / `rot_z_degrees` - the angle of rotation in radians/degrees
+   counter-clockwise about the Cartesian z axis. (tip: the z axis typically
+   faces the camera, and thus will seem like a clockwise rotation)
+
+Rotations are applied in order of z, y, x.
+
+The following table visually explains what positive and negative
+rotations represent.
+|  |  |
```

Comment:
This table is invalid when rendered with Hugo, see https://deploy-preview-121--sourcebots-docs.netlify.com/api/vision/coordinates/#orientation-coordinates

Reply:
Good catch, fixed in eb0f59eb0d0f17c729d00b2e7f314f3f0fd2d2ac